.github/workflows/tests.yml

```diff
@@ -32,3 +32,18 @@ jobs:
           rm db.sqlite3
           python manage.py migrate
           python manage.py test
+
+
+  test_channels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
```

Comment:
You can update this action to v2
```yml
uses: actions/setup-python@v2
```

Reply:
I somehow missed they released v2. Updated